packages/valory/services/apy_estimation/service.yaml

```diff
@@ -23,6 +23,7 @@ models:
         - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
         - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
       history_start: ${SERVICE_APY_HISTORY_START:int:null}
+      history_end: null
```

Comment:
Set the default of the history end to `null` for the service, which resolves to the current timestamp. We need this because we updated the agent's config.